### PR TITLE
fix: zh-cn translate

### PR DESCRIPTION
**Pull request by yayacat**

### CODE REVIEW

File: TREM/index.html [hunk 1]
```diff
@@ -15,13 +15,15 @@
             <span lang="kr">전체화면에서 나갈때는<kbd>F11</kbd>키를 눌러주세요</span>
             <span lang="ru">Press<kbd>F11</kbd>again to exit fullscreen mode.</span>
             <span lang="zh-TW">再次點擊<kbd>F11</kbd>來離開全螢幕模式</span>
+            <span lang="zh-CN">再次点击<kbd>F11</kbd>来离开全萤幕模式</span>
```
Comment (CRooi) [inline TREM/index.html:18]:
```suggest
再次点击<kbd>F11</kbd>来关闭全屏模式

File: TREM/index.html [hunk 2]
```diff
@@ -15,13 +15,15 @@
...
             <span lang="zh-TW">再次點擊<kbd>F11</kbd>來離開全螢幕模式</span>
+            <span lang="zh-CN">再次点击<kbd>F11</kbd>来离开全萤幕模式</span>
         </div>
         <div id="load">
             <div id="app-name" lang="en">Taiwan Real-time Earthquake Monitoring</div>
             <div id="app-name" lang="ja">TREM 台湾リアルタイム地震モニタリング</div>
             <div id="app-name" lang="kr">TREM 대만 실시간 지진 모니터링</div>
             <div id="app-name" lang="ru">TREM Мониторинг землетрясений на Тайване в режиме реального времени</div>
             <div id="app-name" lang="zh-TW">TREM 臺灣即時地震監測</div>
+            <div id="app-name" lang="zh-CN">TREM 台湾及时地震监测</div>
```
Comment (CRooi) [inline TREM/index.html:26]:
```suggest
<div id="app-name" lang="zh-CN">TREM 台湾即时地震监测</div>

File: TREM/index.html [hunk 3]
```diff
@@ -335,6 +350,7 @@
                                                 <span lang="kr">Magnitude</span>
                                                 <span lang="ru">Magnitude</span>
                                                 <span lang="zh-TW">芮氏規模</span>
+                                                <span lang="zh-CN">芮氏规模</span>
```
Comment (CRooi) [inline TREM/index.html:353]:
```suggest
<span lang="zh-CN">里氏震级</span>

**Review by CRooi — CHANGES_REQUESTED** [review]
fix: zh-cn translation

**Review by kamiya10 — APPROVED** [review]

**Pull request merged**